roles/common/tasks/main.yml

```diff
@@ -17,6 +17,17 @@
   apt:
     update_cache: yes
     cache_valid_time: 3600
-  become: yes
-  become_method: sudo
   tags: always
+
+- name: Install dependencies
+  apt:
+    name: '{{ item }}'
+    update_cache: yes
+    cache_valid_time: 3600
+  with_items:
```

Comment:
Move this to name. There's no need to create 6 subtasks, apt module knows how to deal with a list.